.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,24 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/universal
+{
+	"name": "",
+
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "ghcr.io/gpuweb/gpuweb:main",
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	// "features": {},
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	// "forwardPorts": [],
```

Comment:
nit: I'd remove any commented out options that aren't likely to be relevant for us, which I think is all of these. The aka.ms url above has the full documentation if anyone needs it.

Reply:
If we preserve the aka.ms URL, then good to remove. Done!